templates/default/ca-timedrift-metric-script.erb

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+namespace="DECustom"
+region="us-east-1"
+metric_name="CA-timedrift"
+dimension_name="CAName"
+
+ca_list=$(curl -v --user "<%= @ca_stats_user %>:<%= @ca_stats_passwd %>" "<%= @ca_stats_json_url %>" | grep address | awk  -F'"' '{print $4;}' | awk -F. '{print $1;}')
+
+for ca in ${ca_list}
```

Comment:
this runs with the complete list of CAs, including devs.
it's been running during the weekend in the test account.